src/Symfony/Bridge/PhpUnit/SymbolExistenceMock.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit;
+
+/**
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+class SymbolExistenceMock
```

Comment:
maybe call it ClassExistsMock? I know Class is a bit restrictive, but I feel like Symbol is a bit too abstract if we want ppl to understand quickly what's going on, wdyt?

Reply:
Tend to agree yes :+1: is it worth to have 3 classes instead? e.g. `InterfaceExists` + `TraitExists`.

Currently being able to specify the expected type e.g. `'Some\Name' => 'interface'` is missing.

Reply:
I don't think it's needed differenciating between the 3.